s3-file-provider/Dockerfile

```diff
@@ -0,0 +1,86 @@
+# ================================
+# Build image
+# ================================
+FROM swift:5.10-jammy as build
```

Comment:
Shouldn't (new) examples become swift-6 now?

Reply:
Yep that came from the template which I'm just going to update